docs/docs/legal/data-processing-agreement.mdx

```diff
@@ -1,113 +1,276 @@
+
+### Miscellaneous
+
+This DPA shall be governed by and construed in accordance with the governing law and jurisdiction provisions set out in the Agreement, unless required otherwise by Applicable Data Protection Law.
+
+Any liability owed by one party to the other under this DPA shall be subject to the limitations of liability set forth in the Agreement.
+
+This DPA shall terminate upon the earlier of (i) the termination or expiry of all Agreement under which Customer Data may be processed, or (ii) the written agreement of the Parties.
+
+Any notices shall be delivered to a Party in accordance with the notice provisions of the Agreement, unless otherwise specified hereunder.
+
+## Annex 1: Description of Processing Activities / Transfer
+
+### List of Parties
+
+| Data Exporter | Data Importer |
+| :---- | :---- |
+| Name: The Party identified as the Customer in the Agreement. | Name: The Party identified as ZITADEL in the Agreement. |
+| Address:  As identified in the Agreement. | Address: As identified in the Agreement. |
+| Contact Person's Name, position and contact details: As identified in the Agreement. | Contact Person's Name, position and contact details: As identified in the Agreement. |
+| Activities relevant to the transfer:  See below | Activities relevant to the transfer: See below  |
+| Role: Controller  | Role: Processor  |
+
+### Description of processing / transfer:
+
+|  | Description  |
+| :---- | :---- |
+| **Categories of data subjects:** | As described in the section "Processing of Personal Data" of the DPA |
+| **Categories of personal data:**  | As described in the section "Processing of Personal Data" of the DPA |
+| **Sensitive data:** | None. |
+| **If sensitive data, the applied restrictions or safeguards** | N/A |
+| **Frequency of the transfer:** | Continuous |
+| **Nature and subject matter of processing:** | The Services described in the Agreement.   |
+| **Purpose(s) of the data transfer and further processing:**  | As set forth in the Agreement. |
+| **Retention period (or, if not possible to determine, the criteria used to determine that period):** | The personal data may be retained until termination or expiry of the Addendum.   |
 
-### Information and control rights of the customer
+### Competent supervisory authority
 
-The Processor shall provide the Customer with all information necessary to demonstrate compliance with the obligations set forth in this annex or to respond to requests from an applicable supervisory authority, subject to the confidentiality terms in the Framework Agreement.
-The Processor shall enable and contribute to audits, including inspections, carried out by the Customer or an auditor appointed by the Customer.
+The competent supervisory authority in connection with Customer Personal Data protected by the GDPR, is the Hamburg data protection authority. If this is not possible, then as otherwise agreed by the parties consistent with the conditions set forth in Clause 13\.
```

Comment:
```suggestion
The competent supervisory authority in connection with Customer Personal Data protected by the GDPR, is the Hamburg data protection authority. If this is not possible, then as otherwise agreed by the parties consistent with the conditions set forth in Clause 13.
```